src/CoreCompatibilyzer/CoreCompatibilyzer.Runner.NetFramework/CoreCompatibilyzer.Runner.NetFramework.csproj

```diff
@@ -15,8 +15,8 @@
 		<AssemblyTitle>CoreCompatibilyzer.Runner.NetFramework</AssemblyTitle>
 		<AssemblyName>CoreCompatibilyzer.Runner.NetFramework</AssemblyName>
 		<Company>Acumatica</Company>
-		<Copyright>Copyright © 2023 Acumatica Ltd.</Copyright>
-		<Version>1.0.0.0</Version>
+		<Copyright>Copyright © 2025 Acumatica Ltd.</Copyright>
```

Comment:
Acumatica Inc., not Acumatica Ltd.

Reply:
To be more precise, "Acumatica, Inc." (from our official copyright page in doc)